src/types/types.ts

```diff
@@ -1,3 +1,15 @@
+export interface VisitData {
+  type: string
+  displayName: string
+  petNames: string[]
+  startTime: string
+  endTime: string
```

Comment:
shouldnt the time be of type `Date`?

Reply:
Decided to keep as is for now after discussion.